controllers/upgrade.go

```diff
@@ -334,3 +337,56 @@ func (r *RollingUpgradeReconciler) IsScalingGroupDrifted(rollingUpgrade *v1alpha
 	}
 	return false
 }
+
+func (r *RollingUpgradeReconciler) SetInstanceStandBy(instance *autoscaling.Instance, rollingUpgrade *v1alpha1.RollingUpgrade) error {
+	var (
+		instanceID    = aws.StringValue(instance.InstanceId)
+		instanceState = aws.StringValue(instance.LifecycleState)
+	)
+
+	if instanceState != autoscaling.LifecycleStateInService {
+		return nil
+	}
+	r.Info("setting instance to stand-by", "name", rollingUpgrade.NamespacedName(), "instance", instanceID)
+
+	input := &autoscaling.EnterStandbyInput{
+		AutoScalingGroupName:           aws.String(rollingUpgrade.Spec.AsgName),
+		InstanceIds:                    aws.StringSlice([]string{instanceID}),
```

Comment:
Should we do this for the whole batch? i.e. take multiple instances/ids as arguments

Reply:
Also, should we have a wrapper for the AWS call of EnterStandby inside autoscaling.go?

Reply:
Yes, we should move it to autoscaling.go
Regarding sending the whole batch to standby, I think it will get addressed in the next ticket anyway. (parallelize the batch execution). Is that alright? We can also do that for terminate as well.